tests/test_logging.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import argparse
+
+import pytest
+
+from ecl2df.grid import grid_main
+from test_grid import DATAFILE
+
+
+@pytest.mark.parametrize("verbose", [False, True])
+def test_grid_logging(caplog, tmp_path, verbose):
+
+    args = argparse.Namespace(
+        verbose=verbose,
+        DATAFILE=DATAFILE,
+        vectors="*",
+        rstdates="",
+        dropconstants=False,
+        stackdates=False,
+        output=tmp_path / "eclgrid.csv",
+    )
+
+    grid_main(args)
+
+    if verbose:
+        assert caplog.records
```

Comment:
Looks good. You can also assert some strings to be present, like `assert "INFO" in caplog.text`

Reply:
(since any WARNING records could be in `caplog.records` irrespective of verbose (but that should also fail in the `else` below)